ansible/vagrant/README.md

```diff
@@ -20,7 +20,7 @@ vagrant plugin install vagrant-openstack-provider --plugin-version ">= 0.6.1"
 Vagrant uses Ansible to automate the Kubernetes deployment. Install Ansible (Mac OSX example):
 ```
 sudo easy_install pip
-sudo pip install ansible
+sudo pip install ansible==2.0.0.2
```

Comment:
Isn't `sudo pip install ansible` will install latest version which is currently [2.0.0.2](https://pypi.python.org/pypi/ansible/2.0.0.2)? I think it's better to leave `pip install ansible` here.